muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/fi.json

```
@@ -2082,5 +2082,42 @@
       "unsavedWarning": "Tallemattomia muutoksia"
     },
     "notifications": {}
+  },
+  "studymatrix": {
+    "actions": {
+      "cancelSelection": "Peru valinta",
+      "selectOptionalToHops": "Valitse osaksi hopsia",
+      "signUpStudent": "Ilmoita opiskelija",
+      "signupStudentToWorkspace": "Ilmoita työtilaan",
+      "suggested": "Ehdotettu",
+      "suggestOptional": "Ehdota valinnaiseksi",
+      "suggestToNext": "Ehdota seuraavaksi"
+    },
+    "content": {
+      "noSubjectTable": "Ei oppiainetaulukkoa saatavilla kyseiselle opinto-ohjelmalle",
+      "noSuggestionAvailable": "Ei kursseja. Tarkista kurssitarjonta!"
+    },
+    "labels": {
+      "colorDescriptions": "Värien kuvaukset",
+      "completed": "Suoritettu",
+      "courses": "Kurssit",
+      "guidanceCouncelorSuggestion": "Ohjaajan ehdottama",
+      "guidanceCouncelorSuggestionNext": "Ohjaajan seuraavaksi ehdottama",
+      "inprogress": "Kesken",
+      "mandatory": "Pakollinen",
+      "optional": "(*)-Valinnainen",
+      "schoolSubject": "Oppiaine",
+      "selected": "Valittu",
+      "signUpMessage": "Ilmoitusviesti",
+      "signUpStudent": "Ilmoita opiskelija työtilaan",
+      "signUpStudentToWorkspace": "Ilmoita opiskelija työtilaan {{workspaceName}}",
+      "transfered": "Hyväksiluettu",
```

transferred kaikkiin

```suggestion
      "transferred": "Hyväksiluettu",
```